formats.go

```diff
@@ -167,6 +169,32 @@ func isEmail(v interface{}) bool {
 	return err == nil
 }
 
+// isPhone tells whether given string is a valid phone number
+func isPhone(v interface{}) bool {
+	s, ok := v.(string)
+	if !ok {
+		return false
+	}
+
+	// entire phone number to be no more than 15 characters long
```

Comment:
Is this an international standard? If so, where could I look it up? :)

Reply:
Good point - removed this particular check. The library itself is performing it already.

Regarding the standard - according to the [ITU](https://en.wikipedia.org/wiki/International_Telecommunication_Union) the max len for telephone number is 15 digits. But in library that I imported max allowed len is 17 with following [explanation](https://github.com/nyaruka/phonenumbers/blob/b4d8961def2e5317064619bef19ecfb679dc80ee/phonenumbers.go#L23)